.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,8 @@
+Check documentation: https://checkstyle.org/config_xxxxxx.html#XxxxxxXxxxxxXxxxxx
```

Comment:
Such a link is of no use. It is better to add a link to https://checkstyle.org/checks.html where everyone can get to the desired config in one click.

Reply:
such link will not trigger any action from user to provide direct link to Check.
we give template .... user need to fill template

Reply:
What about  https://checkstyle.org/config_[[package name]].html#[[NameOfAffectedCheck]] ?

Reply:
That's better. Now it became clear what should be filled. It is possible that advanced users will be able to fill this out correctly.

 "Name of affected check" is obvious here, but with "package name" users may have problems.

Reply:
Should we place xxxxx as sign of "fill it" ?
Any suggestions ?

Reply:
@strkkk please share your ideas on how to make it clear for user ?

Reply:
@romani There have a been few times where people left `[[]]` as is. Would it make sense to add a statement somewhere to the template to denote `[[]]` must be replaced?

Reply:
Please look at diff, I didnot  use `[[` in URL.

Reply:
> Please look at diff, I didnot use [[ in URL.

I was referring to discussion that wanted to use `[[` in URL.
https://github.com/checkstyle/checkstyle/pull/7327#discussion_r355099402